hmcli/base/scan.py

```diff
@@ -1,10 +1,13 @@
+import click
+
 from rich.console import Console
 from rich.table import Table
 
-from HiveMind_presence import LocalDiscovery
-
+from hmcli.base import hmcli_cmds
 
+@click.command("scan", help="scan for Hives")
```

Comment:
should be scan for Nodes ?

Reply:
Should it?

Reply:
yes, you will find multiple nodes connected to the same hive, individual nodes advertise themselves, not hives

Reply:
But when does a non-Core node advertise itself?

Reply:
for example i added that functionality to the webchat client, its still a local service and a node

i think Node is a more accurate and less confusing terminology

Reply:
I don't follow. I thought the purpose here was to scan for broadcasting nodes in order to connect to them.

Reply:
Which is to say, what's `hmcli scan` gonna show when the webchat answers, and what's the utility?

Reply:
it is identified as its own service instead of HiveMind-websocket. so you (a human) can scan and know what to enter in the browser address

another service i used to have was a REST api, this is all meant for a local network so discovery makes sense